extensions/shared/upgrade-nudge/index.jsx

```diff
@@ -19,14 +19,19 @@ import './store';
 
 import './style.scss';
 
-const UpgradeNudge = ( { autosaveAndRedirectToUpgrade, planName, upgradeUrl } ) => (
+export const UpgradeNudge = ( { autosaveAndRedirectToUpgrade, planName, upgradeUrl } ) => (
 	<Warning
 		actions={
 			// Use upgradeUrl to determine whether or not to display the Upgrade button.
 			// We tried setting autosaveAndRedirectToUpgrade to falsey in `withDispatch`,
 			// but it doesn't seem to be reliably updated after a `withSelect` update.
 			upgradeUrl && [
-				<Button onClick={ autosaveAndRedirectToUpgrade } target="_top" isDefault>
+				<Button
+					href={ upgradeUrl } // Only for server-side rendering, since onClick doesn't work there.
```

Comment:
Do you have thoughts about how could we track this as an event from frontend side? Not a blocker for this PR but just curious if it's possible at all easily.

Reply:
Might not be possible easily :slightly_frowning_face: I was disappointed to find out that `React.renderToStaticMarkup()` actually stripped `onClick` from the element, even when I provided the function I wanted it to call as a string literal.

We might need to resort to attaching it client-side using some DOM method like `addEventListener`.